draft-ietf-v6ops-464xlaton.xml

```diff
@@ -238,7 +248,8 @@ A CLAT instance provides an IPv4 address and the default IPv4 route to the local
 However, the node can also extend the network downstream and provides IPv4 network connectivity to other connected systems.
 Those systems can be physical (e.g. various clients connected to a CPE router), or logical (e.g. virtual systems running on a node, while the host system acts as a router and performs CLAT). 
 In all those cases, systems behind the CLAT node usually use <xref target="RFC1918"/> addresses. 
-The CLAT instance can either translate these IPv4 addresses statelessly to IPv6 using a dedicated IPv6 prefix, or perform a stateful NAT44 between these IPv4 addresses and a dedicated CLAT IPv4 address, which is then statelessly translated to a single dedicated IPv6 address.
+The CLAT instance can either translate these IPv4 addresses statelessly to IPv6 using a dedicated IPv6 prefix as per <xref target="RFC6052"/>, or perform a stateful NAT44 between these IPv4 addresses and a dedicated CLAT IPv4 address, which is then statelessly translated to a single dedicated IPv6 address.
```

Comment:
"can either" when one option is NAT44 gives it a first class status it shouldn't have per the final section of this PR (which I agree with). Should we mention NAT44 down there as an option, or should we remove it from here and elsewhere?

Reply:
I do not think we can remove it really: I believe there are implementations which are doing this, and it's a feasible option (comes with some cost of stateful NAT44, but ...)Until endpoints (like laptops) do not get DHCP-PD support and enterprises start deploying prefix-per-host, it's an only option, really for sharing network connectivity...